index.html

```diff
@@ -71,23 +74,32 @@ <h2>Social Media Marketing</h2>
     <div class="benefits">
       <div>
         <h3>Lead Generation</h3>
-        <img src="./assets/images/lead-generation.png" />
+        <img
+          src="./assets/images/lead-generation.png"
```

Comment:
Not sure if the use of 'representing' is useful in terms of increasing web accessibility.